getting-started/CONCEPTS/ACTIONS.md

```diff
@@ -0,0 +1,61 @@
+# Action
+
+An action is an atomic function that is registered to the AI System. It is a fundamental building block of a plan.
+
+Here's an example of how an action to create a new list would look like in code. Call this action `createList`:
+
+### C#
+
+[List Bot sample](https://github.com/microsoft/teams-ai/blob/a20f8715d3fe81e11c330853e3930e22abe298af/dotnet/samples/04.ai.d.chainedActions.listBot/ListBotActions.cs#L15)
+```C#
+[Action("createList")]
+public bool CreateList([ActionTurnState] ListState turnState, [ActionParameters] Dictionary<string, object> parameters)
```

Comment:
Should we add a ticket to track adding the Python example snippets in the future? (if it doesn't yet exist)

Reply:
Good idea, I'll create a tracking ticket.